UserReportManager.kt

```diff
@@ -1,3 +1,13 @@
 object UserReportManager {
 
+    val userReportService: UserReportService = UserReportService()
+
+    fun sendAllUsersReports(users: List<User>) {
+        for (user in users) {
+            for (userReport in user.reports.orEmpty()) {
```

Comment:
I didn’t think this method was still in use. Is there a reason we chose it here?

Reply:
We use it in a few other places in the app, and the new method doesn't trigger the events that we need in this ticket.

Reply:
We use it in a few other places in the app, and the new method doesn't trigger the events that we need in this ticket.